geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/AlterAsyncEventQueueCommandDUnitTest.java

```diff
@@ -16,6 +16,8 @@
 package org.apache.geode.management.internal.cli.commands;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
```

Comment:
Small nit pick stuff... can you delete these imports and use directly `assertThat(condition).isFalse()` or `assertThat(condition).isTrue()`?. No point in adding a dependency to `org.junit.Assert` if the class already imports `org.assertj.core.api.Assertions.assertThat` (I also think the general consensus is to entirely replace `org.junit.Assert` with `assertj`).